adapters/blip_2/dataloop.json

```diff
@@ -0,0 +1,114 @@
+{
+    "name": "blip-2-huggingface-app",
+    "description": "BLIP-2: Bootstrapping Language-Image Pre-training with Frozen Image Encoders and Large Language Models",
+    "attributes": {
+      "Provider": "Salesforce",
```

Comment:
this should be hugging face no? the other adapters seem to be provided by hugging face

Reply:
Yes correct, the BLIP model also incorrectly identifies the provider as Salesforce